test/integration/api/FactoryAPI.spec.ts

```diff
@@ -101,14 +115,22 @@ describe('FactoryAPI', () => {
     vault = await deployVaultContract(provider);
     core = await deployCoreContract(provider, transferProxy.address, vault.address);
     setTokenFactory = await deploySetTokenFactoryContract(provider, core);
+    rebalancingSetTokenFactory = await deployRebalancingSetTokenFactoryContract(provider, core);
 
     await addAuthorizationAsync(vault, core.address);
     await addAuthorizationAsync(transferProxy, core.address);
 
     assertions = new Assertions(web3, coreWrapper);
 
-    coreWrapper = new CoreWrapper(web3, core.address, transferProxy.address, vault.address);
-    factoryAPI = new FactoryAPI(web3, coreWrapper, assertions, setTokenFactory.address);
+    config = {
+      coreAddress: core.address,
+      transferProxyAddress: transferProxy.address,
+      vaultAddress: vault.address,
+      setTokenFactoryAddress: setTokenFactory.address,
+      rebalancingSetTokenFactoryAddress: rebalancingSetTokenFactory.address,
+    };
+    coreWrapper = new CoreWrapper(web3, config.coreAddress, config.transferProxyAddress, config.vaultAddress);
```

Comment:
Perhaps when we go to clean up later...but at this point feels like it makes more sense to just pass through the config instead of each individual address.